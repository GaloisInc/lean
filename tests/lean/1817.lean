def f : int → int := λ _, 0

#check int.add (-1) 1

#check (-1 : int)

#check 2 + (-1 : int)

#check (2 + -1 : int)

#check f (-1)

#check 2 * (-1 : int)

#check (2 * -1 : int)

#check f 1 < -2

#check (- - 2 : int)
