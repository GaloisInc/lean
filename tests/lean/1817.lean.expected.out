int.add (-1) 1 : ℤ
-1 : ℤ
2 + -1 : ℤ
2 + -1 : ℤ
f (-1) : ℤ
2 * -1 : ℤ
2 * -1 : ℤ
f 1 < -2 : Prop
- -2 : ℤ
